coap/api/transport.go

```diff
@@ -225,21 +223,16 @@ func receive(svc coap.Service, msg *gocoap.Message) *gocoap.Message {
 		return res
 	}
 
-	created, err := ptypes.TimestampProto(time.Now())
-	if err != nil {
-		return nil
-	}
-
-	m := broker.Message{
+	m := mainflux.Message{
 		Channel:   chanID,
 		Subtopic:  subtopic,
 		Publisher: publisher,
 		Protocol:  protocol,
 		Payload:   msg.Payload,
-		Created:   created,
+		Occured:   time.Now().UnixNano(),
```

Comment:
Not sure about this `Occured`... `Created` is very usual in these occasions, but it is true that payload could have been created before (on device). However, `Creted` here relates to the Mainflux `Message{}` entity.

What is the opinion of other @mainflux/maintainers?